flutter_screen_recording_web/lib/flutter_screen_recording_web.dart

```diff
@@ -16,35 +19,107 @@ class WebFlutterScreenRecording extends FlutterScreenRecordingPlatform {
   MediaRecorder mediaRecorder;
   Blob recordedChunks;
   String mimeType;
+  Completer<String> _onStopCompleter;
+  Function(String result) onStop;
+
+  AudioContext _audioContext;
+  MediaStreamAudioDestinationNode _audioDestinationNode;
+
+  //id, source
+  Map<String, MediaStreamAudioSourceNode> _audioSourceNodes;
+
+  WebFlutterScreenRecording();
 
   static registerWith(Registrar registrar) {
     FlutterScreenRecordingPlatform.instance = WebFlutterScreenRecording();
   }
 
   @override
-  Future<bool> startRecordScreen(String name) async {
-    return _record(name, true, false);
+  Future<bool> startRecordScreen(
+    String name, {
```

Comment:
I think we should name it more clearly. What's name ?

Reply:
Ok, I will change that name, please review all the thing so I can fix all at one